src/views/studio/studio-comments.jsx

```diff
@@ -41,11 +41,27 @@ const StudioComments = ({
     handleRestoreComment,
     handleResetComments,
     handleReportComment,
-    handleLoadMoreReplies
+    handleLoadMoreReplies,
+    handleLoadSingleComment
 }) => {
+    const commentHashPrefix = '#comments-';
+    const [singleCommentId, setSingleCommentId] = useState(
+        window.location.hash.indexOf(commentHashPrefix) !== -1 &&
+        parseInt(window.location.hash.replace(commentHashPrefix, ''), 10));
+
     useEffect(() => {
+        if (singleCommentId) {
```

Comment:
I think this `if` should be nested underneath the `comments.length===0 && hasFetchedSession`. The reason being that we only want to fetch that single comment if it hasn't already been fetched. And we only want to load comments after we've `hasFetchedSession`.